pkg/framework/aries/framework_test.go

```diff
@@ -92,11 +90,20 @@ func TestFramework(t *testing.T) {
 			WithKMS(func(ctx api.Provider) (api.CloseableKMS, error) {
 				return &mockkms.CloseableKMS{SignMessageValue: []byte("mockValue")}, nil
 			}),
-			WithCrypter(func(ctx crypto.Provider) (crypto.Crypter, error) {
-				return &didcomm.MockAuthCrypt{EncryptValue: nil}, nil
+			WithDefaultPacker(func(ctx envelope.Provider) (envelope.Packer, error) {
```

Comment:
Is this really "Default" or is it the outbound packer?

I mean the framework puts defaults in for the Packers list too :).

Reply:
Good point, I'll rename.

Reply:
If the outbound packer isn't specified, which packer from `s.packers` is used?

Reply:
Though it is also a default in the sense that it's included in the list of inbound packers, since it makes little sense to send messages using a format you're unwilling to read.

Reply:
If the outbound packer isn't specified, then the framework falls back to initializing the default, which isn't inside `s.packers`. `packager.New` requires a default packager to be provided. I'll add an error check in the constructor if the value is nil.

Reply:
instead of "Default" or "Outbound" it is now just "Packer", since it's used for both in/outbound.
The packer list is now InboundPackers, since they're used inbound only right now. #725 is for making those packers usable outbound as well, by remembering the format to use for a particular connection.